### PR TITLE
Bump to 3.6.15

### PR DESCRIPTION
**Pull request by teake**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

The driver for this PR is https://github.com/python/cpython/pull/27251 (without which Jupyter Lab doesn't run on macOS Big Sur), which has been backported to 3.6.15 according to https://github.com/jupyterlab/jupyterlab/issues/9410#issuecomment-955705925.

### CODE REVIEW

**Comment by teake** [comment]
@conda-forge-admin, please rerender

**Comment by teake** [comment]
> Hmm, I had hoped a simple version bump would suffice, but alas. Looks like the `nis` package doesn't compile on linux64 and aarch64. Any suggestions for fixing this? I must admit I'm a little out of my depth here ...

Adding the `libsnl` dependency seems to have worked :).

File: recipe/meta.yaml [hunk 1]
```diff
@@ -124,6 +124,7 @@ requirements:
     - tk                                 # [unix]
     - ncurses                            # [unix]
     - libffi                             # [unix]
+    - libnsl                             # [linux]
```
Comment (ocefpaf) [inline recipe/meta.yaml:127]: Looks good to me but I'd be more confident if @isuruf could take a quick look before we merge.

Comment (isuruf) [reply recipe/meta.yaml:127]: That's fine, but note that we don't support python 3.6 anymore.

Comment (ocefpaf) [reply recipe/meta.yaml:127]: Yeah. I still think having a new py36 build is good for bugfixes. But building new packages for py36 is a no-no due to CI usage.

**Pull request merged**